.gitignore

```diff
@@ -2,3 +2,4 @@
 !.gitignore
 build/
 out/
+**/*AzureIntegrationTest.java
```

Comment:
Not needed here, right?

Reply:
Well, I have those tests locally. Maybe I should check them in and disable them because they require an actual Azure account

Reply:
Hm, yes, or if it is not checked in, in my opinion, we would not need this line in the repo.